front-end/src/pages/CuisineVote.jsx

```diff
@@ -82,7 +83,7 @@ function CuisineVote() {
           >
           <div id="text">
                 <div className="title-text"> Choose Cuisine</div>
-                <div className="info-text"> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</div>
+                <div className="info-text"> Know exactly what you want? Let your friends know by choosing out of 6 delicious cusine options.</div>
```

Comment:
```suggestion
                <div className="info-text"> Know exactly what you want? Let your friends know by choosing out of 6 delicious cuisine options.</div>
```
Just a small typo (I think)